src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -265,6 +265,100 @@ private async Task<FormulaValue> TryHandleSetProperty(CallNode node, EvalVisitor
 
             return result;
         }
+
+        // Given a TexlFunction, get the implementation to invoke. 
+        private IFunctionInvoker GetInvoker(TexlFunction func)
+        {
+            if (func is IFunctionInvoker invoker)
+            {
+                return invoker;
+            }
+
+            if (func is UserDefinedFunction userDefinedFunc)
+            {
+                return new UserDefinedFunctionAdapter(userDefinedFunc);
+            }
+
+            if (FunctionImplementations.TryGetValue(func, out AsyncFunctionPtr ptr))
+            {
+                return new AsyncFunctionPtrAdapter(ptr);
+            }
+
+            return null;

```

Comment:
>return null; [](http://example.com/codeflow?start=12&length=12)

when is this happening?

Reply:
All the cases we haven't yet converted. See below where it is called and we handle the null.